core/src/main/java/hudson/model/ParametersAction.java

```diff
@@ -89,8 +90,12 @@
 
     private transient Run<?, ?> run;
 
-    public ParametersAction(List<ParameterValue> parameters) {
-        this.parameters = parameters;
+    /**
+     * Constructs a new action with a specified list of parameter values.
+     * @param parameters Parameter values
+     */
+    public ParametersAction(@CheckForNull List<ParameterValue> parameters) {
```

Comment:
I do not think there was ever any intent to support a null method parameter here. I would recommend marking it `@Nonnull` and throwing `NullPointerException`, and if we ever track down the guilty plugin we fix that.